parsl-introduction.ipynb

```diff
@@ -256,7 +256,7 @@
     "# App that echos an input message to an output file\n",
     "@bash_app\n",
     "def slowecho(message, outputs=[]):\n",
-    "    return 'sleep 5; echo %s &> {outputs[0]}' % (message)\n",
+    "    return 'sleep 5; echo %s &> %s' % (message, outputs[0])\n",
```

Comment:
I am adding this because it seems the original version does not work on my end

Reply:
This needs to run through `.format`, not `%`, to work with that keyword substitution I think.